xml/Global.xml

```diff
@@ -104,4 +104,18 @@
   </Panel>
 </VerticalLayout>
 
+<!-- Title Splash when starting a scenario -->
+<Text id="title_splash"
+  position="0 0 0"
```

Comment:
I don't think this is a valid attribute (and should be omitted if this is the default value anyway)

Reply:
this is valid, but it's also the default https://api.tabletopsimulator.com/ui/attributes/#positionsize-attributes-advanced